README.md

```diff
@@ -27,12 +27,24 @@ The grape-swagger gem provides an autogenerated documentation for your [Grape](h
 
 These screenshot is based on the [Hussars](https://github.com/LeFnord/hussars) sample app.
 
+## Model Parsers
+
+Since 0.21.0, `Grape::Entity` is not a part of grape-swagger, you need to add `grape-swagger-entity` manually to your Gemfile:
```

Comment:
Separate `grape-swagger-representable` and add something like "Grape-swagger also supports Representable. To ... add ...".